src/utils.js

```diff
@@ -143,15 +155,14 @@ export function standardizeWitUnits(
     const num = new BigNumber(amount)
     const exponent = witUnitConversor[inputCurrency][outputCurrency]
     const result = num.times(new BigNumber(10).pow(exponent), 0)
-
     if (truncate === -1) {
-      return result.toString()
+      return new BigNumber(result).toFixed()
```

Comment:
```suggestion
      return result
```

Reply:
Wait, result is already a BigNumber? Then it should be 
```suggestion
      return result.toFixed()
```